src/components/StorageOverview/Utilization/Utilization.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { UtilizationBody } from '../../Dashboard/Utilization/UtilizationBody';
+import { UtilizationItem } from '../../Dashboard/Utilization/UtilizationItem';
+import { getUtilizationVectorStats } from '../../../selectors';
+import { formatBytes } from '../../../utils';
+
+export const Utilization = ({ iopsUtilization, latencyUtilization, throughputUtilization, LoadingComponent }) => {
+  const iopsStats = getUtilizationVectorStats(iopsUtilization);
+  const latencyStats = getUtilizationVectorStats(latencyUtilization);
+
+  const throughputStatsRaw = getUtilizationVectorStats(throughputUtilization);
+  let throughputStats = null;
+  let throughputMax = 0;
+  let throughputMaxConverted;
+  if (throughputStatsRaw) {
+    throughputMax = Math.max(0, ...throughputStatsRaw);
```

Comment:
@mareklibra can you please look at the max value here? If it is visualization friendly?